pkg/addons/data/DaemonSet.apps/openshift-azure-monitoring/etcd-metrics.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    k8s-app: etcd-metrics
+  name: etcd-metrics
+  namespace: openshift-azure-monitoring
+spec:
+  selector:
+    matchLabels:
+      k8s-app: etcd-metrics
+  strategy:
+    rollingUpdate:
+      maxSurge: 0
+  template:
+    metadata:
+      labels:
+        k8s-app: etcd-metrics
+    spec:
+      containers:
+      - args:
+        - -loglevel
+        - debug
+        - -hostname
+        - https://$(HOSTNAME):2379
```

Comment:
this works?

Reply:
yes

Reply:
TIL (or maybe today I remembered) https://github.com/kubernetes/kubernetes/pull/8149